index.ts

```diff
@@ -83,16 +98,21 @@ async function main() {
     m4337: await m4337Deployment(getSafe4337ModuleDeployment),
     moduleSetup: await m4337Deployment(getSafeModuleSetupDeployment),
     entryPoint: new ethers.Contract(
-      "0x0000000071727De22E5E9d8BAf0edAc6f37da032",
+      // "0x0000000071727De22E5E9d8BAf0edAc6f37da032",
+      "0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789",
```

Comment:
This entry point is not supported by the module. You should use a bundler that supports the latest (v0.7) entry point.

Reply:
The two bundles I tried only returned this address as a supported entry point. I would love to know where to find an aggregated list of providers.

Reply:
I know that Pimlico supports `0x0000000071727De22E5E9d8BAf0edAc6f37da032`. That is what I used when we were hacking together, and it accepted the user op.

Reply:
Looks like I may have to change the format of this entry point:

```
EntryPoint 0x0000000071727De22E5E9d8BAf0edAc6f37da032
Error: {"message":"UserOperation mismatch, EntryPoint expects UserOperation of version 0.7","code":-32602}
```

Reply:
> Now we are being rejected with Signature Error...

This is expected as long as the signature is not valid.

One thing that I forgot to do is re-encode the validity in the signature when sending the user operation. Specifically: https://github.com/bh2smith/nearly-safe/pull/1/files#r1638437917

Reply:
Cool beans. Will try it out again later tonight. 

In the mean time. You can create a testnet wallet at 

https://testnet.wallet.mintbase.xyz/

Export your private key put it in the env along with your account id

Here is a faucet: 

https://near-faucet.io/

You will also have to fund the safe you wind up with.